### ISSUE TITLE
Case sensitive changelog file check

### ISSUE THREAD

**Issue by expipiplus1** [lspitzner/iridium#1]
Does it make sense to check if any changelog file exists, and not just 'ChangeLog.md'?

**Comment by lspitzner** [comment]
Yes indeed; in general the creation of a initial config needs to be "gather-info >>= generate-initial" instead of "pure static-default-config". E.g. auto-detect if there is a git repo etc.
